build.gradle

```diff
@@ -72,7 +72,7 @@ ext {
 }
 
 dependencies {
-    implementation 'junit:junit:${versions.junit}'
+    implementation 'junit:junit:4.13.1'
```

Comment:
We shouldn't modify this.

Reply:
it will report error when run "./gradlew build"

Reply:
Did you rebase the code on main branch?

Reply:
yes, of course, my code is forked from the latest code under the main branch

Reply:
rollback